pkg/controller/storagecluster/reconcile.go

```diff
@@ -626,23 +666,32 @@ func (r *ReconcileStorageCluster) ensureCephConfig(sc *ocsv1.StorageCluster, req
 // ensureCephCluster ensures that a CephCluster resource exists with its Spec in
 // the desired state.
 func (r *ReconcileStorageCluster) ensureCephCluster(sc *ocsv1.StorageCluster, reqLogger logr.Logger) error {
-	// if StorageClass is "gp2" or "io1" based, set tuneSlowDeviceClass to true
-	// this is for performance optimization of slow device class
-	//TODO: If for a StorageDeviceSet there is a separate metadata pvc template, check for StorageClass of data pvc template only
-	for i, ds := range sc.Spec.StorageDeviceSets {
-		throttle, err := r.throttleStorageDevices(*ds.DataPVCTemplate.Spec.StorageClassName)
-		if err != nil {
-			return fmt.Errorf("Failed to verify StorageClass provisioner. %+v", err)
-		}
-		if throttle {
-			sc.Spec.StorageDeviceSets[i].Config.TuneSlowDeviceClass = true
-		} else {
-			sc.Spec.StorageDeviceSets[i].Config.TuneSlowDeviceClass = false
+	if !sc.Spec.ExternalStorage.Enable {
+		// if StorageClass is "gp2" or "io1" based, set tuneSlowDeviceClass to true
+		// this is for performance optimization of slow device class
+		//TODO: If for a StorageDeviceSet there is a separate metadata pvc template, check for StorageClass of data pvc template only
+		for i, ds := range sc.Spec.StorageDeviceSets {
+			throttle, err := r.throttleStorageDevices(*ds.DataPVCTemplate.Spec.StorageClassName)
+			if err != nil {
+				return fmt.Errorf("Failed to verify StorageClass provisioner. %+v", err)
+			}
+			if throttle {
+				sc.Spec.StorageDeviceSets[i].Config.TuneSlowDeviceClass = true
+			} else {
+				sc.Spec.StorageDeviceSets[i].Config.TuneSlowDeviceClass = false
+			}
 		}
 	}
 
+	var cephCluster *cephv1.CephCluster
+	var cephClusterTypeStr = "CephCluster"
```

Comment:
Similar to StorageClusters, there is no such thing as a new external CephCluster CR. I think it's fine to have at least one message saying that we're creating an external cluster, but we shouldn't need to keep repeating it for future log messages.

Reply:
Done...